content/chat/rooms/typing.textile

```diff
@@ -28,19 +28,19 @@ The following is the structure of a typing event:
 
 ```[json]
 {
-  "currentlyTyping": [
+  "currentlyTyping": {
     "clemons",
     "zoranges",
-  ],
+  },
 }
 ```
 
 The following are the properties of a typing event:
 
 |_. Property |_. Description |_. Type |
-| currentlyTyping | An array of all users currently typing. | Array |
+| currentlyTyping | A set of all users currently typing. | Set |
 
-You can use the length of the @currentlyTyping@ array to decide whether to display individual user names, or that multiple people are typing in your user interface.
+You can use the length of the @currentlyTyping@ set to decide whether to display individual user names, or that multiple people are typing in your user interface.
```

Comment:
size of the set. Annoyingly, sets have sizes and arrays have lengths.

Reply:
Ah thanks. Fixed!